server/server.go

```diff
@@ -84,6 +87,31 @@ func Server(ver string) *chi.Mux {
 	if err != nil {
 		logger.Error("failed to read/process configuration", zap.Error(err))
 	}
+	if conf.LogPath != "" {
```

Comment:
Since, there is a default value defined for LogPath. This check should be update so that logs can be logged to console.

Reply:
makes sense i will work on it , to summarise 
we need to output the logs in the console , in case when empty string is passed in env , this would over write  the default value of `"logfile"`

Reply:
We need logs to be piped in file when there is a path provided, else by default logs should be printed on console. 
That's why default value of LogPath should be empty string.